IMC_WhyReact/blog.md

```diff
@@ -0,0 +1,357 @@
+# LINK TO MY LOOM VIDEO:
+<a href='https://www.loom.com/share/c085de33c3fa4e9e8d495f91c9cb71ad'>Click me!</a>
+
+<br />
+
+<div align="center">
+  <img src="./img/0.png" />
+</div>
+
+# What's with all the craze about React?
```

Comment:
This is great! Just need a few lines of description for each of the coding examples at the end. The code is commented well, but your readers might not look into the code when they first check out your blogs. Should be ready after adding those descriptions!

Reply:
@joeyhuaa I'm also not able to see the images in the blog itself - can you confirm if this is just on my end?